src/actions/mod.rs

```diff
@@ -9,6 +9,7 @@ pub use self::delete_bucket::DeleteBucket;
 pub use self::delete_object::DeleteObject;
 #[cfg(feature = "full")]
 pub use self::delete_objects::{DeleteObjects, ObjectIdentifier};
+pub use self::get_bucket_policy::{GetBucketPolicy, GetBucketPolicyResponse};
```

Comment:
This needs to be gated on the `full` feature as it requires dependencies that are only available when it's enabled. You can check if it compiles afterwards with `cargo check --no-default-features`